src/dialogs/more-info/controls/more-info-time.ts

```diff
@@ -14,15 +14,15 @@ class MoreInfoTime extends LitElement {
   @property({ attribute: false }) public stateObj?: HassEntity;
 
   protected render() {
-    if (!this.stateObj || isUnavailableState(this.stateObj.state)) {
+    if (!this.stateObj || this.stateObj.state === UNAVAILABLE) {
       return nothing;
     }
 
     return html`
       <ha-time-input
         .value=${this.stateObj.state}
```

Comment:
we also set the value of the input to `unknown`, we should probably guard for that as it is not a valid time?

Reply:
I changed it to set `undefined` which means the time/date widgets are empty which is the correct visual state from my POV.